internal/action/sync.go

```diff
@@ -20,11 +21,12 @@ import (
 	"github.com/gopasspw/gopass/pkg/ctxutil"
 	"github.com/gopasspw/gopass/pkg/debug"
 	"github.com/urfave/cli/v2"
+	"github.com/xhit/go-str2duration"
 )
 
 var (
-	autosyncIntervalDays = 3
-	autosyncLastRun      time.Time
+	autosyncInterval = time.Duration(3*24) * time.Hour
+	autosyncLastRun  time.Time
 )
 
 func init() {
```

Comment:
Should the GOPASS_AUTOSYNC_INTERVAL be given the same treatment? Or we dont care since its deprecated?

Reply:
Would be nice to handle it in the same way, but not strictly necessary since it's deprecated.

Reply:
Let's not do that, forgoing complexity.